spring-integration-core/src/main/java/org/springframework/integration/support/management/ExponentialMovingAverage.java

```diff
@@ -66,18 +66,9 @@ public synchronized void reset() {
 	/**
 	 * Add a new measurement to the series.
 	 *
-	 * @param value the measurement to append (milliseconds)
+	 * @param value the measurement to append
 	 */
 	public synchronized void append(double value) {
-		appendNanos(value * 1000000);
```

Comment:
Well, according to the EMA description (http://www.onlinetradingconcepts.com/TechnicalAnalysis/MAExponential.html) (and others in the Internet)  we really don't get deal with nanos :smile: .
That is just our case that we collect send/handle "duration" values for EMA like traders do with prices or currency rates.